.golangci.example.yml

```diff
@@ -389,10 +389,17 @@ linters-settings:
     settings:
       mnd:
         # the list of enabled checks, see https://github.com/tommy-muehle/go-mnd/#checks for description.
-        checks: argument,case,condition,operation,return,assign
-        # ignored-numbers: 1000
-        # ignored-files: magic_.*.go
-        # ignored-functions: math.*
+        checks:
+        - argument
+        - case
+        - condition
+        - operation
+        - return
+        - assign
```

Comment:
its also possible to supply it as comma-separated string, however having an array as example better from a documentation perspective (easier to add validation checks for `json schema`)